cdap-ui/app/features/admin/controllers/namespace/templates-list-ctrl.js

```diff
@@ -0,0 +1,29 @@
+angular.module(PKG.name + '.feature.admin')
+  .controller('NamespaceTemplatesListController', function ($scope, mySettings, $stateParams, $alert) {
+
+    var vm = this;
+
+    mySettings.get('pluginTemplates')
+      .then(function (res) {
+        vm.list = res[$stateParams.nsadmin];
+      });
+
+    vm.delete = function (template) {
+      delete template;
+      mySettings.get('pluginTemplates')
+        .then(function (res) {
+          delete res[$stateParams.nsadmin][template.templateName];
```

Comment:
What if we start out with empty plugin templates? 
```res``` will be undefined and undefined[...][...] will be a js error right?